docs/release-notes/8.5.asciidoc

```diff
@@ -25,6 +39,13 @@ There are no user-facing changes in 8.5.2.
 [[release-notes-8.5.1]]
 === 8.5.1
 
+[discrete]
+[[known-issue-8.5.1]]
+==== Known issues
+* The rule details page and **Edit rule settings** page load indefinitely if you edit a rule that has the `saved_id` property configured. All rule types, except for the custom query rule, are affected.
++
+Upgrading to 8.6 or higher resolves this issue. If you’d prefer to stay on 8.5, use the Update rule API to remove the `saved_id` field from the non-functioning `query`, `eql`, `machine_learning`, `threat_match`, `threshold`, or `new_terms` rule.
```

Comment:
This is a good idea, @vitaliidm! Something to note: I'd need to link to the main page (i.e., the page title) because the "Field required for saved-query rules `PUT` calls" section doesn't have an anchor. 

```suggestion
Upgrading to 8.6 or higher resolves this issue. If you’d prefer to stay on 8.5, use the Update rule API to remove the `saved_id` field from the non-functioning `query`, `eql`, `machine_learning`, `threat_match`, `threshold`, or `new_terms` rule.
```

We also have the option of using the direct URL of the section, which is what you provided in your comment above. I'd personally advise against doing this since there's always the risk of the link breaking. Anchors tend to be more stable and easier for us to track in terms of link maintaince.

Reply:
sounds good to me! Thanks for the suggestion